sonaranalyzer-dotnet/src/SonarAnalyzer.Common/Helpers/SymbolHelper.cs

```diff
@@ -237,13 +237,15 @@ public static Accessibility GetEffectiveAccessibility(this ISymbol symbol)
 
             for (var container = symbol.ContainingType; container != null; container = container.ContainingType)
             {
-                switch (container.DeclaredAccessibility)
+                if (container.DeclaredAccessibility == Accessibility.Private)
                 {
-                    case Accessibility.Private:
-                        return Accessibility.Private;
-                    case Accessibility.Internal:
-                        result = Accessibility.Internal;
-                        continue;
+                    return Accessibility.Private;
+                }
+
+                if (container.DeclaredAccessibility == Accessibility.Internal)
+                {
+                    result = Accessibility.Internal;
+                    continue;

```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this redundant jump. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=csharpsquid%3AS3626)